overlay/libexec/k3os/boot

```diff
@@ -117,6 +117,23 @@ setup_hostname()
 
 setup_mounts()
 {
+
+    for x in $(cat /proc/cmdline); do
+         case $x in
+            k3os.install.iso_url=*)
+                INSTALL_ISO_URL=${x#k3os.install.iso_url=}
+                ;;
+         esac
+    done
+
+    if [ -v INSTALL_ISO_URL ]; then
+        pinfo NETBOOT Activating eth0 to download ${INSTALL_ISO_URL}
+        dhclient eth0
```

Comment:
@metahertz is the `dhclient eth0` required if you are booting from the network? The interface should be setup already, no?

Reply:
Hey @dweomer in my testing, eth0 was down so the wget failed, if there is a better way to "assume" the network details given to iPXE by DHCP instead of doing DHCP again within the initrd i'm not aware of it (but 100% would be cleaner).

Reply:
IMHO netbooting only gets you so far to do the netboot itself, which runs through iPXE or the network card's firmware, and loads something. After that a basic Linux is getting started (vmlinuz & initrd), but at that point, you have no network anymore|yet.